### PR TITLE
add gifrop recipe

### PR DESCRIPTION
**Pull request by Jtrachsel**
Adds recipe for gifrop.  

Genomic islands from roary pangenomes.

Pipeline to extract genomic islands or, units of horizontal gene transfer, from roary pangenomes.

Bacterial genomes only.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by Jtrachsel** [comment]
@BiocondaBot please add label

File: recipes/gifrop/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+ host:
+  - abricate=1.0.0
+  - parallel
+  - roary=3.13.0
+  - prokka=1.14.6
+  - r-base=3.6.3
```
Comment (dpryan79) [inline recipes/gifrop/meta.yaml:24]: Does it have to be exactly 3.6.3? Is there any reason the 4.0 won't be supported?

File: recipes/gifrop/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+ host:
```
Comment (dpryan79) [inline recipes/gifrop/meta.yaml:19]: Please remove the `host:` section, it isn't needed for this recipe.

Comment (Jtrachsel) [reply recipes/gifrop/meta.yaml:19]: Thanks for looking at this so quickly.  I was unable to get this to build without at least a host entry for abricate.  I have two custom databases that need to be set up. I set up these databases in the build script by using the output of the `abricate --list` command to put the files in the correct location.  Is there a better way of doing this?

File: recipes/gifrop/meta.yaml (hunk 1 above)
Comment (Jtrachsel) [reply recipes/gifrop/meta.yaml]: I changed these to be the minimum version requirements

File: recipes/gifrop/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/gifrop/meta.yaml:19]: That's totally fine.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**